tests/base/test_regularization.py

```diff
@@ -302,5 +303,35 @@ def test_update_of_sparse_norms(self):
         self.assertTrue(reg.objfcts[2].norm == 1.)
         self.assertTrue(reg.objfcts[3].norm == 1.)
 
+    def test_linked_properties(self):
+        mesh = Mesh.TensorMesh([8, 7, 6])
+        reg = Regularization.Tikhonov(mesh)
+
+        [self.assertTrue(reg.regmesh is fct.regmesh) for fct in reg.objfcts]
+        [self.assertTrue(reg.mapping is fct.mapping) for fct in reg.objfcts]
+
+        D = reg.regmesh.cellDiffx
+        reg.regmesh._cellDiffx = 4*D
```

Comment:
@fourndo: this tests a use-case similar to the one that you showed with updating the cell Diffs